packages/codemods/jest.config.js

```diff
@@ -0,0 +1,8 @@
+const { createConfig } = require('@uifabric/build/jest/jest-resources');
+
+const config = createConfig({
+  testMatch: ['<rootDir>/src/**/__tests__/test/**/*.[jt]s?(x)', '<rootDir>/src/**/?(*.)+(spec|test).[tj]s?(x)'],
+  testRegex: undefined,
```

Comment:
Not sure you need to override this--all your tests appear to be named `*.test.ts`, which should be picked up by the default glob. (Or consider fixing the default glob if it's wrong.)

Reply:
I definitely need to override this. At least since I'm not using .test because of 
![image](https://user-images.githubusercontent.com/20801176/86650151-4cf03900-bf97-11ea-98a4-9c3fcbf49a98.png)

It looks like it will run all files under __test__ OR those that have .test. Once I switch to just a test folder I can do away with this.

Reply:
Hmm okay. Can you use just `testRegex` though? And with a glob that only reflects your specific tests.